pkg/overlay/cache_test.go

```diff
@@ -98,6 +98,20 @@ func testCache(ctx context.Context, t *testing.T, store storage.KeyValueStore, s
 			assert.Error(t, err)
 		}
 	}
+
+	{ // Delete
+		// Test standard delete
+		err := cache.Delete(ctx, valid1ID)
```

Comment:
nit - also test that `cache.Get` returns an error for `valid1ID` after this assertion

Reply:
and maybe that it does not return an error before this assertion

Reply:
Test added 👍